### DISCUSSION TITLE
Terminates after a message

### DISCUSSION THREAD

**Discussion by ravishnu17** [outline/outline#5698] [Self Hosting] [answered]
It runs smoothly for path one month. Now It returns like this? What I have to do now?

{"label":"lifecycle","level":"info","message":"\u001b[32m\u001b[39m\n\u001b[32mIs your team enjoying Outline? Consider supporting future development by sponsoring the project:\u001b[39m\n\u001b[32m\u001b[39m\n\u001b[32mhttps://github.com/sponsors/outline\u001b[39m\n\u001b[32m\u001b[39m"}

Killed

**Comment by tommoor** [comment, marked as answer]
This means that the process was killed rather than exited cleanly. Something on your server killed the process outside of Outlines control
